### ISSUE TITLE
Form validation error message issue

### ISSUE THREAD

**Issue by dsamojlenko** [cds-snc/c19-benefits-node#211]
> If you don’t select a radio button and try to go to the next screen, you get returned to the same page, but no error message is given.  The error message should be appended to the `<legend>`.
